web-local/configs/config-ddl-local-dc2.yaml

```diff
@@ -35,11 +35,11 @@ systemOfRecord:
                      
 databus:
    keyspaces:
-      databusdc2:
```

Comment:
why do we change this?

Reply:
b/c with this keyspace, dc2 won't start.